### DISCUSSION TITLE
Instalación del django

### DISCUSSION THREAD

**Discussion by diego20366** [frlp-utn-ingsoft/catedra#16] [Q&A] [answered]
Hola profes, ¿cómo les va?
Estuve buscando en el cvg algún foro pero no lo encontré, entonces les consulto por aca.
Tengo instalado el git, y python 3.12 pero no me deja instalar el Django. Cuando sigo los pasos que dice el tutorial, en el python me tira "invalid syntax".
No les consulté en la clase pasada porque tenía instalado el git y el python pero todavía no había intentado instalar el django.
No sé si me falta un paso hacer, o que estoy haciendo mal.
Lo intente hacer en el powershell de windows y tambien me tira error.
Desde ya muchas gracias.
Saludos
![Screenshot_1](https://github.com/frlp-utn-ingsoft/catedra/assets/62917982/af82aa47-ddf6-47a4-aa4d-560524fce1ad)

![Screenshot_1](https://github.com/frlp-utn-ingsoft/catedra/assets/62917982/f37f2683-9ae2-4054-a98f-90bf94130955)

**Comment by RodrigoJacznik** [comment, marked as answer]
Para instalar Django lo tenes que hacer desde el powershell, no desde el repl de python.
El error dice que no encuentra py en el path. Fijate de agregarlo, te dejo un link https://realpython.com/add-python-to-path/#how-to-add-python-to-path-on-windows.

**Reply by RodrigoJacznik** [reply]
Al instalar python es importante marcar esta casilla.

![image](https://github.com/frlp-utn-ingsoft/catedra/assets/2715743/7378d4ae-cb2a-4de9-bced-e8b3baf5fb38)
